tests/unit/manage/test_forms.py

```diff
@@ -69,10 +69,53 @@ def test_validate_role_name_fails(self, value, expected):
 class TestAddEmailForm:
     def test_creation(self):
         user_service = pretend.stub()
-        form = forms.AddEmailForm(user_service=user_service)
+        form = forms.AddEmailForm(user_service=user_service, user_id=pretend.stub())
 
         assert form.user_service is user_service
 
+    def test_email_exists_error(self):
+        user_id = pretend.stub()
+        form = forms.AddEmailForm(
+            data={"email": "foo@bar.com"},
+            user_id=user_id,
+            user_service=pretend.stub(find_userid_by_email=lambda _: user_id),
+        )
+
+        assert not form.validate()
+        assert (
+            form.email.errors.pop()
+            == "This email address is already being used by this account. "
```

Comment:
Maybe we could assign the error message in a variable i.e. `expected_error` and then do the assertion like this
```python
assert from.email.errors.pop() == expected_error
```
I think that'd improve readability a bit :)

Reply:
I essentially tried to write something consistent with other tests, especially with those in `./tests/unit/accounts/test_forms.py`; see, e.g.: https://github.com/pypa/warehouse/blob/3db5abc1c80a98658fa97b6953c545bc6aacf740/tests/unit/accounts/test_forms.py#L311

I can change that if needed, though.

Reply:
Oh, thanks for noticing that. 
And you're right, we should prioritize consistency around the codebase!

LGTM 👍